iXBRLViewerPlugin/viewer/src/js/outline.test.js

```diff
@@ -351,21 +351,21 @@ describe("Section grouping", () => {
 
         // Make f1c hidden.  We now have ELR1*2 ELR3*1 ELR1*2
         // The first ELR1 run should be selected.
-        report.getItemById("f1c").ixNode.wrapperNode = $('');
+        report.getItemById("f1c").ixNode.isHidden = true;
```

Comment:
It would be good to have a test that sets different combinations of `wrapperNode` and `isHidden` and ensures that `isHidden()` returns the correct value.

It would be good to have a test that `_preProcessiXBRL` now populates `wrapperNode`.

Reply:
I added this scenario to the test document document1.html in the samples folder. Outside of that I do not see an established way to mock a JQuery object to test the changes in viewer.js. 

Perhaps @paulwarren-wk has some ideas on this otherwise we will have to rely on manual testing to verify that we do not cause any regressions.